master/security/adopt-zero-trust.md

```diff
@@ -0,0 +1,310 @@
+
+The {{site.prodname}} control plane handles distributing all the policy information from the {{site.prodname}} data store to each enforcement point, ensuring that all network connections are subject to enforcement (Requirement 4). It translates the high-level declarative policy into the detailed enforcement attributes that change as applications scale up and down to meet demand, and evolve as developers modify them.
+
+##### Istio Citadel Identity System
+
+In {{site.prodname}} and Istio, workload identities are based on Kubernetes Service Accounts. An Istio component called Citadel handles minting cryptographic keys for each Service Account to prove its identity on the network (Requirement 2) and encrypt traffic (Requirement 5). This allows the Zero Trust Network to be resilient even if attackers compromise network infrastructure like routers or links.
+
+### How to
+
+This section explains how to establish a Zero Trust Network using {{site.prodname}} and Istio. It is written from the perspective of platform and security engineers, but should also be useful for individual developers looking to understand the process.
+
+Building and maintaining a Zero Trust Network is the job of an entire application delivery organization, that is, everyone involved in delivering a networked application to its end users. This includes:
+
+- Developers, DevOps, and Operators
+- Platform Engineers
+- Network Engineers
+- Security Engineers and Security Operatives
+
+In particular, the view that developers build applications which they hand off to others to figure out how to secure is incompatible with a Zero Trust Network strategy. In order to function correctly, a Zero Trust Network needs to be configured with detailed information about expected flows---information that developers are in a unique position to know.
+
+At a high level, you will undertake the following steps to establish a Zero Trust Network:
+
+1. Install {{site.prodname}}.
+1. Install Istio and enable {{site.prodname}} integration.
+1. Establish workload identity by using Service Accounts.
+1. Write initial whitelist policies for each service.
+
+After your Zero Trust Network is established, you will need to maintain it. 
+
+#### Install {{site.prodname}}
+
+Follow the [install instructions]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/) to get {{site.prodname}} software running in your cluster.
+
+#### Install Istio and enable {{site.prodname}} integration
+
+Follow the instructions to [Enable Application Layer Policy]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/app-layer-policy).
+
+The instructions include a “demo” install of Istio for quickly testing out functionality. For a production installation to support a Zero Trust Network, you should instead follow the official Istio install instructions. Be sure to enable mutually authenticated TLS (mTLS) in your install options by setting **global.mtls.enabled to true**.
```

Comment:
Did you intend to have only `global.mtls.enabled` bolded?

Reply:
I meant for all to be bold, for easy scanning, but it would be fine if you think just the setting should be highlighted.